client/src/components/Modals/index.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import Modal from 'react-bootstrap/Modal';
+import Button from 'react-bootstrap/Button';
+import "./style.css";
+import axios from 'axios';
+
+class MyVerticallyCenteredModal extends React.Component {
+  constructor(props) {
+    super(props)
+    this.logout = this.logout.bind(this)
+    this.state = {
+      modalShow: true
+    }
+  }
+
+  logout(event) {
+    event.preventDefault()
+    console.log('logging out')
+    axios.post('/logout').then(response => {
+      console.log(response.data)
+      if (response.status === 200) {
+        this.props.updateUser({
+          loggedIn: false,
+          username: null
+        })
+      }
+    }).catch(error => {
+      console.log('Logout error')
```

Comment:
Need better handling of the error just console logging the text 'logout error isnt really showing me what the error is in the console. Bare minimum should be console.log("Logout Error: ", error); What happens when i catch an error? Should i return from the function or will this function not return? Am i supposed to be expecting something coming back from this? Confused on why you would put a catch on updating a User to null and not logged in. Is props.UpdateUser just going to switch the states?